### ISSUE TITLE
Error with 560Ti

### ISSUE THREAD

**Issue by lauren-elise-thomas** [xmrig/xmrig-nvidia#40]
./xmrig-nvidia -o ca.minexmr.com:4444 -u 4B78kerqaWTWjsUVb7xvmVcRkPfruTr7SLFoCDZek6j3f8jtyDXkvSFNxEDZgZ8NXZ8FPK5pCXghuTmz8vnLHeLfHYZs82u.gpu --donate-level=5
 * VERSIONS:     XMRig/2.4.2 libuv/1.8.0 CUDA/7.50 gcc/5.4.0
 * CPU:                 Intel(R) Core(TM) i5-3570K CPU @ 3.40GHz x64 AES-NI
 * GPU #0:       GeForce GTX 760 @ 1150/3004 MHz 52x18 0x0 arch:30 SMX:6
 * GPU #1:       GeForce GTX 560 Ti @ 1701/2052 MHz 30x16 0x0 arch:21 SMX:8
 * ALGO:         cryptonight, donate=5%
 * POOL #1:      ca.minexmr.com:4444
 * COMMANDS:     hashrate, health, pause, resume
[CUDA] Error gpu 1: <cryptonight_extra_cpu_init>:193 "out of memory"
terminate called after throwing an instance of 'std::runtime_error'
  what():  [CUDA] Error: out of memory
Aborted (core dumped)

**Comment by crazydoc72** [comment]
Try to lower blocks_x_threads from 30x16 to 15x16 in your config.json and see if this resolves the issue.

**Comment by tomassrnka** [comment]
Hi, I'm having the same issue, adding "block_x_threads": "15x16" into json config does not help

**Comment by crazydoc72** [comment]
No, this is not the correct syntax :-D. Check config.json and you will find the correct one!
